test/erc721m.test.ts

```diff
@@ -357,7 +359,7 @@ describe('ERC721M', function () {
     it('revert if contract is not mintable', async () => {
       await contract.setStages([
         {
-          price: ethers.utils.parseEther('0.5'),
+          price: ethers.utils.parseEther('0.1'),
```

Comment:
What's the reason for switching all these 0.5 to 0.1?

Reply:
I confronted some strange "no sufficient ether to send" issue for some tests after I removing files. For short-term bandage, I lowered the price in test.

Reply:
Makes sense!